app/components/UI/Perps/Views/PerpsMarketDetailsView/PerpsMarketDetailsView.tsx

```diff
@@ -479,6 +496,17 @@ const PerpsMarketDetailsView: React.FC<PerpsMarketDetailsViewProps> = () => {
         />
       )}
 
+      {isEligibilityModalVisible && (
+        <PerpsBottomSheetTooltip
+          isVisible
+          onClose={() => setIsEligibilityModalVisible(false)}
+          contentKey={'geo_block'}
```

Comment:
Is this a constant that can be used elsewhere in the app? If not it's probably fine to leave a hard coded string

```suggestion
          contentKey='geo_block'
```